config/master.key

```diff
@@ -0,0 +1 @@
+b5deca5a9e5c49fed852c289b57cd185
```

Comment:
Pls don't it shouldn't be committed, but added to gitignore and stored in a safe place like 1password